{"name": "test_unsuccessful_otp_send_to_invalid_email", "status": "passed", "steps": [{"name": "This test verifies that OTP does not send to an invalid email", "status": "passed", "start": 1648799147980, "stop": 1648799148184}], "start": 1648799147979, "stop": 1648799148184, "uuid": "1d8641a3-116b-4e48-95af-e74c524ae460", "historyId": "965ea806611f8025d243ae1fac785daf", "testCaseId": "b691a4a0ed760957e87d93347b60160c", "fullName": "tests.api_testcases.verify_email.verify_email_test.TestVerifyEmail#test_unsuccessful_otp_send_to_invalid_email", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.verify_email"}, {"name": "suite", "value": "verify_email_test"}, {"name": "subSuite", "value": "TestVerifyEmail"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "108970-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.verify_email.verify_email_test"}]}